content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,87 @@
+
+I think we should start with the traffic Excalidraw received. We ended the year at **20k weekly active users** which is pretty incredible for a project that was started only a year ago.
+
+https://excalidraw.com/#json=6443031091740672,amxJZJxlZAlUBLADWIukFg
+
+### Commits
+
+This success was only made possible thanks to the over [100 contributors](https://github.com/excalidraw/excalidraw/graphs/contributors) that helped us ship new features, improvements and fixes right to production.
+
+https://excalidraw.com/#json=5035223658004480,t4Y53hm-6O1PgeUVwG_GMQ
+
+## Tech stack
+
+Excalidraw is fully open source, but it also stands on the shoulders of many other projects. We couldn't do it without [Rough.js](https://github.com/rough-stuff/rough), the library that gives Excalidraw its unique look, and [React](https://github.com/facebook/react), of course. [TypeScript](https://github.com/microsoft/TypeScript) is also essential to tame the complexity of our codebase, giving a helping to hand to new contributors (and old ones alike). Our full list of [dependencies](https://github.com/excalidraw/excalidraw/network/dependencies) is listed on GitHub.
+
+In addition, Excalidraw was made possible thanks to awesome services such as [Vercel](https://vercel.com/) for hosting and pull request previews, [Crowdin](https://crowdin.com/project/excalidraw) for managing dozens of translation, [Code Sandbox](https://codesandbox.io/) to get started hacking on the project, [Sentry](https://sentry.io/) for error reporting, and [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) to keep our dependencies up to date.
+
+## Some of our coolest features
+
+### 🤝 Collaboration
+
+When the lockdown started, companies all around the world struggled to adapt to remote work. [idlewinn](https://github.com/idlewinn) and [petehunt](https://github.com/petehunt) [implemented live collaboration](https://github.com/excalidraw/excalidraw/pull/879) in Excalidraw to enable it to be used for interviews, brainstorms, presentations, and more. If you're curious, we [explained the inner workings in a blog post](/building-excalidraw-p2p-collaboration-feature/).
+
+### 🔒 Your data is encrypted
+
+Most of Excalidraw use cases involve drawing sensitive data, we architected our system such that our servers never see the content of your drawings through end-to-end encryption. The [backend](https://github.com/excalidraw/excalidraw-json) support for storing data was implemented by [lipis](https://github.com/lipis) and the client [support for encrypting](https://github.com/excalidraw/excalidraw/pull/642) the data before submitting was done by [vjeux](https://github.com/vjeux). Read more on how we are doing it in [end to end encryption](/end-to-end-encryption/) article.
```

Comment:
```suggestion
Many of Excalidraw use cases involve drawing sensitive data. As such, we architected our system so that our servers never see the content of your drawings, using end-to-end encryption. The [backend](https://github.com/excalidraw/excalidraw-json) support for storing data was implemented by [lipis](https://github.com/lipis), while the [client-side encryption itself](https://github.com/excalidraw/excalidraw/pull/642) was added by [vjeux](https://github.com/vjeux). Read  how we are doing it in our article on [end to end encryption](/end-to-end-encryption/).
```